config.json

```diff
@@ -1,64 +1 @@
-{
```

Comment:
Why do you need to change config.json? If you've not made any specific changes, please revert it.

Reply:
changes made